http/handler.go

```diff
@@ -117,6 +122,25 @@ func (i Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	i.corsHandler.ServeHTTP(w, r)
 }
 
+// Init returns an instance of Jaeger Tracer that samples 100% of traces and logs all spans to stdout.
+func InitGlobalTracer(service string) (opentracing.Tracer, io.Closer) {
```

Comment:
Need to find a more appropriate location for this method, possibly add it to go-log?